parlai/crowdsourcing/tasks/acute_eval/README.md

```diff
@@ -76,29 +74,29 @@ Note that we assume that "dialogue" consists of strictly alternating turns (e.g.
 
-By setting `extra_args['onboarding_threshold']`, you can also adjust the minimum proportion of onboarding tasks (if you have multiple) that must be answered correctly to pass onboarding.
+By setting `onboarding_threshold`, you can also adjust the minimum proportion of onboarding tasks (if you have multiple) that must be answered correctly to pass onboarding.
 
 
 ## Other settings
 
 ### Task configuration on MTurk
 
-The title, description, and keywords of the task as shown on MTurk default to values in `ARG_STRING` in `example_script.py`. These values are used as follows:
-- `--task-title`: A short and descriptive title about the kind of task that the HIT contains. On the Amazon Mechanical Turk web site, the HIT title appears in search results and everywhere that the HIT is mentioned.
-- `--task-description`: Includes detailed information about the kind of task that the HIT contains. On the Amazon Mechanical Turk web site, the HIT description appears in the expanded view of search results, and in the HIT and assignment screens.
-- `--task-tags`: One or more words or phrases that describe the HIT, separated by commas. On MTurk website, these words are used in searches to find HITs.
-- `--additional-task-description`: Additional text to show in the left-hand pane of the chat window.
+The title, description, and keywords of the task are set by values in `conf/example.yaml`. These values are used as follows:
+- `task_title`: A short and descriptive title about the kind of task that the HIT contains. On the Amazon Mechanical Turk web site, the HIT title appears in search results and everywhere that the HIT is mentioned.
+- `task_description`: Includes detailed information about the kind of task that the HIT contains. On the Amazon Mechanical Turk web site, the HIT description appears in the expanded view of search results, and in the HIT and assignment screens.
+- `task_tags`: One or more words or phrases that describe the HIT, separated by commas. On MTurk website, these words are used in searches to find HITs.
+- `additional_task_description`: Additional text to show in the left-hand pane of the chat window.
 
 
-### CLI arguments
+### YAML and CLI arguments
 
-A comprehensive list of settings specific to ACUTE-Eval can be found in `add_args_to_group()` in `acute_eval_blueprint.py`. For the arguments most likely to be useful for running ACUTE-Eval, see `example_script.py`.
+A comprehensive list of settings specific to ACUTE-Eval can be found in `AcuteEvalBlueprintArgs` in `acute_eval_blueprint.py`. For examples of how these arguments can be set in practice, see `conf/example.yaml`. For modifying the `task_reward` argument (for example) on the command line, append `mephisto.task.task_reward=${DESIRED_TASK_REWARD}`.
```

Comment:
I'd note as well that people can specify their own yaml files in `conf/<anything>.yaml` and then use those instead of `example` with `conf=<anything>`.

Reply:
Great, added